xcom/BUILD

```diff
@@ -0,0 +1,67 @@
+java_library(
+	name = "migration-lib",
+	srcs = ["Migration.java"],
+	deps = [
+		"@graknlabs_client_java//:client-java",
+		"@graknlabs_graql//java:graql",
+		"@maven//:org_sharegov_mjson",
+        "@maven//:com_univocity_univocity_parsers"
```

Comment:
Missing indent

Reply:
Ah you caught an issue there - the xcom project uses tabs, not spaces, because my editor settings weren't aligned with the rest of the codebase.
Fixed throughout all files.